### PR TITLE
docs(aio): update myUnless references to appUnless

### PR DESCRIPTION
**Pull request by hartraft**
fixes Issue Number: #20447

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
myUnless is referenced in the documentation where appUnless is shown in code examples

Issue Number: #20447 

## What is the new behavior?
appUnless is referenced in the aio docs

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by kapunahelewong** [comment]
@hartraft You even did the e2e tests! Thanks!

@gkalpak LGTM, is there anything else I should do to help this along?

**Review by kapunahelewong — APPROVED** [review]

File: aio/content/guide/structural-directives.md [hunk 1]
```diff
@@ -685,10 +685,10 @@ You inject both in the directive constructor as private variables of the class.
 
 
 
-### The _myUnless_ property
+### The _appUnless_ property
 
-The directive consumer expects to bind a true/false condition to `[myUnless]`.
-That means the directive needs a `myUnless` property, decorated with `@Input`
+The directive consumer expects to bind a true/false condition to `[appUnless]`.
+That means the directive needs a `appUnless` property, decorated with `@Input`
```
Comment (gkalpak) [inline aio/content/guide/structural-directives.md]: _a_ --> _an_

File: aio/content/guide/structural-directives.md [hunk 2]
```diff
@@ -708,16 +708,16 @@ Read about `@Input` in the [_Template Syntax_](guide/template-syntax#inputs-outp
 
 
 
-Angular sets the  `myUnless` property whenever the value of the condition changes.
-Because the `myUnless` property does work, it needs a setter.
+Angular sets the  `appUnless` property whenever the value of the condition changes.
```
Comment (gkalpak) [inline aio/content/guide/structural-directives.md]: Unrelated to this PR, but while you are at it, feel free to fix the double-space :grin:

**Review by gkalpak — CHANGES_REQUESTED** [review]
I left a couple of minor comments.
Can you also squash the commits together?

**Comment by hartraft** [comment]
@gkalpak thanks for reviewing the PR! I've made the changes needed 😃

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
The Travis failures are not related to this PR. ("Inherited" from master.)

**Pull request closed without merging**